src/components/Modals/BlockUnblockUserModal/BlockUnblockUserModal.tsx

```diff
@@ -84,7 +84,10 @@ const BlockUnblockUserModal = ({
     };
 
     const blockUnblockError = errorMessages.find(
-        error => error.code === ERROR_CODES.PERMISSION_DENIED || error.code === ERROR_CODES.INVALID_ADVERTISER_ID
+        error =>
+            error.code === ERROR_CODES.PERMISSION_DENIED ||
+            error.code === ERROR_CODES.INVALID_ADVERTISER_ID ||
+            error.code === ERROR_CODES.ADVERTISER_NOT_REGISTERED
```

Comment:
check for this edge case when user is not verified

![Screenshot 2024-11-08 at 3 58 53 PM](https://github.com/user-attachments/assets/0b54c0f9-187a-40f1-8841-502defba936b)